src/dfmt/formatter.d

```diff
@@ -803,15 +813,34 @@ private:
 
     void formatRightBrace()
     {
-        if (astInformation.structInitEndLocations.canFindIndex(tokens[index].index))
+        void popToBeginIndent(LocationPair locPair)
```

Comment:
It looks like this is trying to solve the same problem that `indentToMostRecent` on the indent stack should be solving.

Reply:
oh it was not oblivious from the code of from naming